src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/loadingScripts.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Helper function to load scripts.
+ *
+ * It runs only once because of the visualization implementation : file is loaded once, then
+ * the onDataReceived() is called multiple times, so it won't load the same script on this or any
+ * other visualization show/hide.
+ */
+function loadScript(url) {
+    let script = document.createElement("script");
+    script.src = url;
+
+    document.head.appendChild(script);
+}
+
+/**
+ * Helper function to load styles.
+ *
+ * It runs only once because of the visualization implementation : file is loaded once, then
+ * the onDataReceived() is called multiple times, so it won't load the same style on this or any
+ * other visualization show/hide.
+ */
```

Comment:
I don't think this doc is correct here. Nothing in implementation of this function prevents multiple style loading, so this doc can bring confusion and false thinking that we will be safe to call it multiple times with the same style source. The same applies to doc comment above